src/api/public/apidocs/paths/source_project_name_package_name_view_info.yaml

```diff
@@ -10,19 +10,11 @@ get:
       name: view
       schema:
         type: string
+        # There are further view points, but this documentation is just for view=info
```

Comment:
Why shouldn't we document, or at least keep, the rest of the options? I see you removed these options:
- cpio
- getmultibuild
- issues
- products
- productrepositories

I see they are still being used in the backend. See:
- https://github.com/openSUSE/open-build-service/blob/54cd549ce822914f182a0cda2c381b56254a6195/src/backend/bs_srcserver#L7894-L7896
- https://github.com/openSUSE/open-build-service/blob/54cd549ce822914f182a0cda2c381b56254a6195/src/backend/bs_srcserver#L2992
- https://github.com/openSUSE/open-build-service/blob/54cd549ce822914f182a0cda2c381b56254a6195/src/backend/bs_srcserver#L3005

Reply:
because the cgi parameter set are only valid for this view.

Reply:
Documenting these two endpoints (`view` being `info` and `view` being other values) in different entries is a compromise between the different outputs returned by the same endpoint.

These lines were added to make a reference between each other endpoint in the documentation:
1. the one you propose to remove, and
2. this other one (see [this endpoint](https://api.opensuse.org/apidocs/index#/Sources%20-%20Files/get_source__project_name___package_name_) and scroll down until the `view` parameter):
https://github.com/openSUSE/open-build-service/blob/cbd8fce77d6d2b18dbbe495d55bf5688d67d2c12/src/api/public/apidocs/paths/source_project_name_package_name.yaml#L98

This makes it easier for documentation users to navigate between both endpoints.

I can understand repeating the values in `enum` section could be redundant. I'm fine if [those lines](https://github.com/openSUSE/open-build-service/blob/cbd8fce77d6d2b18dbbe495d55bf5688d67d2c12/src/api/public/apidocs/paths/source_project_name_package_name_view_info.yaml#L14-L19) are removed.

But please, don't remove the reference in the description field. I helps API users find and relate the documentation they are looking for. By the way, I have never heard from any API user complaining about too much, repetitive, or redundant API documentation.